system-tests/src/test/java/org/eclipse/edc/samples/transfer/transfer00prerequisites/Transfer00prerequisitesTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright (c) 2023 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH - Initial implementation
+ *
+ */
+
+package org.eclipse.edc.samples.transfer.transfer00prerequisites;
```

Comment:
I wouldn't add a package for every sample, instead, a package `common` containing the common logic would be more meaningful